scipy/sparse/linalg/tests/test_special_sparse_arrays.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import numpy as np
+from numpy.testing import assert_array_equal, assert_allclose
+
+from scipy.sparse import diags, csgraph
+from scipy.linalg import eigh
+
+from scipy.sparse.linalg import LaplacianNd
+
+
+class TestLaplacianNd:
+    """
+    LaplacianNd tests
+    """
+    INT_DTYPES = [np.int8, np.int16, np.int32, np.int64]
+    REAL_DTYPES = [np.float32, np.float64]
+    COMPLEX_DTYPES = [np.complex64, np.complex128]
+    ALLDTYPES = INT_DTYPES + REAL_DTYPES + COMPLEX_DTYPES
+
+    @pytest.mark.parametrize("bc", ['neumann', 'dirichlet', 'periodic'])
+    def test_1d_specific_shape(self, bc):
+        bc = "neumann"
```

Comment:
This test is broken, in the sense that it overwrites the `bc` passed through parametrization with a constant value

```suggestion
```

Reply:
thanks! Sorry to miss this evident bug...